src/components/sponsor/sponsorBanner.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import styles from "./sponsorBanner.module.css"
+import emailIcon from "../../assets/images/icons/email-white-144px.png"
+import downIcon from "../../assets/images/icons/down-button.png"
+import { Link, animateScroll as scroll } from "react-scroll"
+
+
+const HeroBanner = () => (
+  <section className={styles.hero}>
+    <div className={styles.heroContainer}>
+      <div className={styles.heroTextArea}>
+        <h1 className={styles.title}>
+          Sponsorship
+        </h1>
+        <p className={styles.subtext}>
+          The generous support we receive from our sponsors enables us to continue pursuing our mission;
+          to provide a welcoming and inclusive experience that complements the academic components of 
+          studying an Engineering degree at the University of Auckland.
+        </p>
+        <div className={styles.heroBtnContainer}>
+          <Link
```

Comment:
It is possible to implement this without a library. I did it for the 2020 launch - but it was kinda dodgy because linking to a page from a component is technically a bit dodgy (the component could theoretically be used on a different page, meaning that the link would be dead).
Realistically we solve this by tidying up our 3 different banners (lol), that's a problem for this summer.